### ISSUE TITLE
Android crashes on startup

### ISSUE THREAD

**Issue by leovanhaaren** [Instabug/Instabug-Flutter#77]
Hi there,

I've integrated Instabug according to the guide on both Android and iOS. iOS works fine but android is crashing the app on startup.

**Stacktrace**
```
2019-06-14 10:02:26.658 23225-23225/com.example.my_app E/AndroidRuntime: FATAL EXCEPTION: main
    Process: com.example.my_app, PID: 23225
    io.reactivex.exceptions.CompositeException: 2 exceptions occurred. 
        at io.reactivex.internal.observers.LambdaObserver.onError(LambdaObserver.java:80)
        at io.reactivex.internal.observers.LambdaObserver.onNext(LambdaObserver.java:67)
        at io.reactivex.internal.operators.observable.ObservableSubscribeOn$SubscribeOnObserver.onNext(ObservableSubscribeOn.java:58)
        at io.reactivex.internal.operators.observable.ObservableObserveOn$ObserveOnObserver.drainNormal(ObservableObserveOn.java:200)
        at io.reactivex.internal.operators.observable.ObservableObserveOn$ObserveOnObserver.run(ObservableObserveOn.java:252)
        at io.reactivex.android.schedulers.HandlerScheduler$ScheduledRunnable.run(HandlerScheduler.java:109)
        at android.os.Handler.handleCallback(Handler.java:873)
        at android.os.Handler.dispatchMessage(Handler.java:99)
        at android.os.Looper.loop(Looper.java:193)
        at android.app.ActivityThread.main(ActivityThread.java:6669)
        at java.lang.reflect.Method.invoke(Native Method)
        at com.android.internal.os.RuntimeInit$MethodAndArgsCaller.run(RuntimeInit.java:493)
        at com.android.internal.os.ZygoteInit.main(ZygoteInit.java:858)
      ComposedException 1 :
    	java.lang.ArrayIndexOutOfBoundsException: length=0; index=0
        at com.instabug.library.instacapture.b.a(InstaCapture.java:102)
        at com.instabug.library.instacapture.b.b(InstaCapture.java:30)
        at com.instabug.library.instacapture.b$1.a(InstaCapture.java:118)
        at com.instabug.library.instacapture.b$1.accept(InstaCapture.java:110)
        at io.reactivex.internal.observers.LambdaObserver.onNext(LambdaObserver.java:63)
        at io.reactivex.internal.operators.observable.ObservableSubscribeOn$SubscribeOnObserver.onNext(ObservableSubscribeOn.java:58)
        at io.reactivex.internal.operators.observable.ObservableObserveOn$ObserveOnObserver.drainNormal(ObservableObserveOn.java:200)
        at io.reactivex.internal.operators.observable.ObservableObserveOn$ObserveOnObserver.run(ObservableObserveOn.java:252)
        at io.reactivex.android.schedulers.HandlerScheduler$ScheduledRunnable.run(HandlerScheduler.java:109)
        at android.os.Handler.handleCallback(Handler.java:873)
        at android.os.Handler.dispatchMessage(Handler.java:99)
        at android.os.Looper.loop(Looper.java:193)
        at android.app.ActivityThread.main(ActivityThread.java:6669)
        at java.lang.reflect.Method.invoke(Native Method)
        at com.android.internal.os.RuntimeInit$MethodAndArgsCaller.run(RuntimeInit.java:493)
        at com.android.internal.os.ZygoteInit.main(ZygoteInit.java:858)
     Caused by: java.lang.ArrayIndexOutOfBoundsException: length=0; index=0
        at com.instabug.library.instacapture.b.a(InstaCapture.java:102)
        at com.instabug.library.instacapture.b.b(InstaCapture.java:30)
        at com.instabug.library.instacapture.b$2.a(InstaCapture.java:130)
        at com.instabug.library.instacapture.b$2.accept(InstaCapture.java:120)
        at io.reactivex.internal.observers.LambdaObserver.onError(LambdaObserver.java:77)
        at io.reactivex.internal.observers.LambdaObserver.onNext(LambdaObserver.java:67)
        at io.reactivex.internal.operators.observable.ObservableSubscribeOn$SubscribeOnObserver.onNext(ObservableSubscribeOn.java:58)
        at io.reactivex.internal.operators.observable.ObservableObserveOn$ObserveOnObserver.drainNormal(ObservableObserveOn.java:200)
        at io.reactivex.internal.operators.observable.ObservableObserveOn$ObserveOnObserver.run(ObservableObserveOn.java:252)
        at io.reactivex.android.schedulers.HandlerScheduler$ScheduledRunnable.run(HandlerScheduler.java:109)
        at android.os.Handler.handleCallback(Handler.java:873)
        at android.os.Handler.dispatchMessage(Handler.java:99)
        at android.os.Looper.loop(Looper.java:193)
        at android.app.ActivityThread.main(ActivityThread.java:6669)
        at java.lang.reflect.Method.invoke(Native Method)
        at com.android.internal.os.RuntimeInit$MethodAndArgsCaller.run(RuntimeInit.java:493)
        at com.android.internal.os.ZygoteInit.main(ZygoteInit.java:858)
      ComposedException 2 :
2019-06-14 10:02:26.659 23225-23225/com.example.my_app E/AndroidRuntime: 	java.lang.ArrayIndexOutOfBoundsException: length=0; index=0
        at com.instabug.library.instacapture.b.a(InstaCapture.java:102)
        at com.instabug.library.instacapture.b.b(InstaCapture.java:30)
        at com.instabug.library.instacapture.b$2.a(InstaCapture.java:130)
        at com.instabug.library.instacapture.b$2.accept(InstaCapture.java:120)
        at io.reactivex.internal.observers.LambdaObserver.onError(LambdaObserver.java:77)
        at io.reactivex.internal.observers.LambdaObserver.onNext(LambdaObserver.java:67)
        at io.reactivex.internal.operators.observable.ObservableSubscribeOn$SubscribeOnObserver.onNext(ObservableSubscribeOn.java:58)
        at io.reactivex.internal.operators.observable.ObservableObserveOn$ObserveOnObserver.drainNormal(ObservableObserveOn.java:200)
        at io.reactivex.internal.operators.observable.ObservableObserveOn$ObserveOnObserver.run(ObservableObserveOn.java:252)
        at io.reactivex.android.schedulers.HandlerScheduler$ScheduledRunnable.run(HandlerScheduler.java:109)
        at android.os.Handler.handleCallback(Handler.java:873)
        at android.os.Handler.dispatchMessage(Handler.java:99)
        at android.os.Looper.loop(Looper.java:193)
        at android.app.ActivityThread.main(ActivityThread.java:6669)
        at java.lang.reflect.Method.invoke(Native Method)
        at com.android.internal.os.RuntimeInit$MethodAndArgsCaller.run(RuntimeInit.java:493)
        at com.android.internal.os.ZygoteInit.main(ZygoteInit.java:858)
```

**DEVICE**
samsung SM-G950F

**OS**
OS Level 28

**Comment by salmamali** [comment]
@leovanhaaren We're looking into this and we'll keep you updated.

**Comment by salmamali** [comment]
@leovanhaaren You can update now, this was fixed in version 1.0.0-beta.4. Feel free to reopen the issue if the problem persists.
